redisson/src/main/java/org/redisson/RedissonSetCache.java

```diff
@@ -1273,6 +1273,11 @@ public int addAllIfAbsent(Map<V, Duration> objects) {
         return get(addAllIfAbsentAsync(objects));
     }
 
+    @Override
+    public boolean addIfAbsent(Map<V, Duration> objects) {
```

Comment:
This method should be added RSetCacheReactive and RSetCacheRx interfaces as well.

Reply:
> This method should be added to RSetCacheReactive and RSetCacheRx interfaces as well.

OK，it has been added, please check.